hash/mo5_flop.xml

```diff
@@ -850,10 +850,11 @@ Thanks to DCMOTO (http://dcmoto.free.fr) for info!
 	</software>
 
 	<software name="mandrago">
-		<description>Mandragore (Hacked by Prehisto)</description>
+		<description>Mandragore (hacked)</description>
 		<year>1984</year>
 		<publisher>Infogrames</publisher>
-
+		<info name="hacked" value="Prehisto" />
+		
```

Comment:
Why does it show a diff for the blank line here?  Is there trailing whitespace?

Reply:
Looks like github being weird, doesn't appear in https://github.com/mamedev/mame/pull/10125.patch

Reply:
> Why does it show a diff for the blank line here? Is there trailing whitespace?

You're right. Blank spaces were left behind.
I removed them.
Now it's OK.